### PR TITLE
Feature/Django 3.1.1 and fixes postgres issue with password

### PR DESCRIPTION
**Pull request by jdrb2098**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,10 +13,14 @@ services:
       - .:/src
       - ./scripts:/scripts
     # Dev user: admin pass: admin
-    command: ["/scripts/wait-for-it.sh", "db:5432", "--", "/scripts/command-dev.sh"]
+    # command: ["/scripts/wait-for-it.sh", "db:5432", "--", "/scripts/command-dev.sh"]
+    command: ["/bin/bash", "/scripts/command-dev.sh"]
+
```
Comment (EdisonArango) [inline docker-compose.yml]: Please config back the previous command

File: docker-compose.yml [hunk 2]
```diff
@@ -14,9 +14,13 @@ services:
       - ./scripts:/scripts
     # Dev user: admin pass: admin
     command: ["/scripts/wait-for-it.sh", "db:5432", "--", "/scripts/command-dev.sh"]
+    
+
```
Comment (EdisonArango) [inline docker-compose.yml]: Why did you add these spaces?

**Comment by jdrb2098** [comment]
Ohh sorry, i don't even notice... I guess I just need to be more careful

File: docker-compose.yml (hunk 2 above)
Comment (EdisonArango) [reply docker-compose.yml]: Please remove them

Comment (EdisonArango) [inline docker-compose.yml:18]: ```suggestion```

Comment (EdisonArango) [inline docker-compose.yml]:
```suggestion
```
